coderd/database/migrations/000119_workspace_proxy_name_idx.down.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+
+DROP INDEX IF EXISTS workspace_proxies_lower_name_idx;
+
+-- Enforces no active proxies have the same name.
+CREATE UNIQUE INDEX ON workspace_proxies (name) WHERE deleted = FALSE;
```

Comment:
shouldn't it be `CREATE UNIQUE INDEX workspace_proxies_name_idx ON workspace_proxies USING btree (name) WHERE deleted = FALSE;`?

Reply:
Nop, I copied the previous migration that added the index: https://github.com/coder/coder/blob/9e218f73dfb630df1c094d0dd152902ef3de4f57/coderd/database/migrations/000114_workspace_proxy.up.sql#L20-L21

Forgot to use a btree there, so kept it as it was for the Down.